website/docs/reference/resource-configs/trino-configs.md

```diff
@@ -18,11 +17,13 @@ In some specific cases, there may be need for tuning during the Trino session pr
 }}
 ```
 
-### Use table properties to configure connector specifics
+## Connector properties
+
+You can use Starburst/Trino table properties to configure how you want your data to be represented.
```

Comment:
this section felt weird, and your attempt to fix it made it clearer to me. basically it's a caveat / "heads up", that there are many more possible configurations that can be set in the model config, and to learn what they are you should look at the docs

for example there's this option for Hive specifically:
`hive.metastore.thrift.delete-files-on-drop=true`


basically what I'm trying to say in this section is:
> there are dozens of connectors, and for each connector there are lots of specifc settings tweaks you may want to do, so:
> 1. go read up on them, and
> 2. if you find one, you can set it within the `properties` key of a model config object